### ISSUE TITLE
User without sub-model permissions denied access

### ISSUE THREAD

**Issue by jon-nfc** [nofusscomputing/centurion_erp_ui#39]
i.e. when viewing a device, if the users is missing device-software permission they are denied access from viewing the device.

## Links

- Blocked By: nofusscomputing/centurion_erp#442

## Requirements

- [ ] User can view primary model
- [ ] Child/Secondary model does not show

**Comment by jon-nfc** [comment]
closing as no longer relevant
